docs/docs/creating-global-styles.md

```diff
@@ -2,7 +2,124 @@
 title: Creating global styles
 ---
 
-In this guide, we'll walk through using traditional global styles in your site.
+In nearly every site, there will be some global styles, such as a reset or typography defaults. This guide will walk through how to add global styles to your site, whether you use standard `.css` files or a CSS-in-JS solution.
```

Comment:
standard css files or w/ preprocessor. Probably should clarify this as preprocessors are so common.